app/models/interest.rb

```diff
@@ -1,3 +1,5 @@
 class Interest < ApplicationRecord
-  belongs_to :user
+  self.inheritance_column = :_type_disabled
+  enum type: { health: 0, hobby: 1, work: 2 }
+  belongs_to :user, optional: true
```

Comment:
If interest have to has ```user_id``` so it is not optional. This option will allow creating ```Interest``` without ```user_id```, it is not what we want to achieve.